pkg/controller/import-controller_test.go

```diff
@@ -685,6 +685,38 @@ var _ = Describe("Update PVC from POD", func() {
 		Expect(resPvc.GetAnnotations()[AnnVddkVersion]).To(Equal("1.0.0"))
 	})
 
+	It("Should delete pod for scratch space even if retainAfterCompletion is set", func() {
```

Comment:
I don't see the retainAfterCompletion annotation set on the PVC

Reply:
I have added this to the annotations list and made it easier to read.